convert/src/vertex_to_bd/text_to_badgerdoc_use_case.py

```diff
@@ -0,0 +1,74 @@
+import tempfile
+from pathlib import Path
+
+from .badgerdoc_format.badgerdoc_format import BadgerdocFormat
+from .badgerdoc_format.pdf_renderer import PDFRenderer
+from .badgerdoc_format.plain_text_converter import (
+    PlainTextToBadgerdocTokenConverter,
+)
+from .models.common import S3Path
+
+
+class TextToBDConvertUseCase:
```

Comment:
May you make class name more "elegant"?

Reply:
https://github.com/epam/badgerdoc/issues/170